### PR TITLE
fixed `visualize_monthly_changes_in_hirakund_reservoir_using_video.ipynb`

### PR DESCRIPTION
**Pull request by tarunkukreja003**
resolves [#9791](https://github.com/ArcGIS/geosaurus/issues/9791)
works with AGOL

### CODE REVIEW

File: samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_hirakund_reservoir_using_video.ipynb [hunk 1]
```diff
@@ -60,25 +60,1240 @@
    "cell_type": "code",
```
Comment (ManushiM) [inline samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_hirakund_reservoir_using_video.ipynb:60]: <p>Is there a particular reason you stopped execution of this cell?</p><p> </p><p>If you don't need to install, leave the cell as is without running it, and add a markdown note above saying that 'If you do not have imageio you may install it as follows, else proceed to the cell following that'.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1747/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (tarunkukreja003) [reply samples/04_gis_analysts_data_scientists/visualize_monthly_changes_in_hirakund_reservoir_using_video.ipynb:60]: <p>thanks for catching this, corrected</p>

**Review by jyaistMap — CHANGES_REQUESTED** [review]
* Please remove the output from the _!conda install_ commands - it isn't necessary for the guide, and will be different for everyone who runs it.  There are also alarming warning cells that don't help with the guide. It also makes the guide very long to scroll through.

**Comment by tarunkukreja003** [comment]
> * Please remove the output from the _!conda install_ commands - it isn't necessary for the guide, and will be different for everyone who runs it.  There are also alarming warning cells that don't help with the guide. It also makes the guide very long to scroll through.

complete

**Review by jyaistMap — APPROVED** [review]
* built website locally, looks great

**Pull request merged**